front-end/src/AdminBar/AdminView.js

```diff
@@ -54,28 +57,40 @@ class AdminView extends React.Component {
   }
 
   render() {
+    const { typed } = this.state;
+    const newNumber = !this.props.numberList.includes(typed);
```

Comment:
Nice! I'd consider moving this logic to a method, e.g.
```
isNewNumber = number = !this.props.numberList.includes(number);
```

Also, how much of are we polyfilling? Array#includes is not supported in all browsers :) If not, we can use `Array#indexOf(i) > -1` :)

Reply:
Hmm i had no idea it wasn't supported! I think we can trust dsek not use use IE or Opera mobile, so i don't think we need to consider those, goog catch though! :)

Reply:
moved the logic to method btw :)